core/container/src/main/resources/modules/org/wildfly/swarm/container/runtime/module.xml

```diff
@@ -49,10 +49,7 @@
     <module name="org.jboss.as.controller" export="true"/>
     <module name="org.jboss.as.self-contained"/>
     <module name="org.jboss.shrinkwrap"/>
-    <module name="org.jboss.weld.api" slot="3" export="true"/>
-    <module name="org.jboss.weld.spi" slot="3" export="true"/>
-    <module name="org.jboss.weld.core" slot="3"/>
-    <module name="org.jboss.weld.se" slot="3"/>
+    <module name="org.jboss.weld.se"/>
```

Comment:
Why does this only get replaced with `org.jboss.weld.se` but the next file gets the modules replaced like for like?